.gitignore

```diff
@@ -3,3 +3,5 @@
 battlesnake
 .idea
 .vscode
+env
+venv
```

Comment:
Is leaving a newline at the end of the line convention? Github is flagging this.